R/sim_gs_n.R

```diff
@@ -360,6 +360,7 @@ sim_gs_n <- function(
 #' # Cut the trial data
 #' cutting(trial_data)
 create_cut <- function(...) {
+  lapply(X = list(...), FUN = force)
```

Comment:
Just a quick technical note: I'm not sure if everyone knows that `force()` is simply semantic sugar (as its help page `?force` says). All the sugar  helps with is perhaps code readability. Technically, if you want to avoid the lazy/delayed evaluation of function arguments, you can simply use the arguments earlier by placing their names where you want them to be evaluated, e.g., here you can simply write `list(...)` instead of `lapply(X = list(...), FUN = force)`.

Reply:
Personally I like the increased readability of `force()`. As it is, this isn't a widely known feature. Also I added a comment in #262 to explain the purpose. I'd be open to switching to the minimal `list(...)` once the comment is available for context

Reply:
Either way is fine to me. I don't have a strong opinion. Personally I slightly prefer low calories :)